src/EuroDNS/Helper/EuroDNSApi.php

```diff
@@ -0,0 +1,1793 @@
+            <contact:address1>' . $registrantDetails['address1'] . '</contact:address1>
+            <contact:address2>' . $registrantDetails['state'] . '</contact:address2>
+            <contact:city>' . $registrantDetails['city'] . '</contact:city>
+            <contact:zipcode>' . $registrantDetails['postcode'] . '</contact:zipcode>
+            <contact:country_code>' . $registrantDetails['country_code'] . '</contact:country_code>
+            <contact:email>' . $registrantDetails['email'] . '</contact:email>
+            <contact:phone>' . $contactPhone . '</contact:phone>
+            <contact:fax></contact:fax>
+        </contact:' . $action . '>';
+
+        return $contact;
+    }
+
+    /***
+     * Function to split the name into firstname and last name based on the space between them
+     * if there is no speration then take firstname and last name as same
+     */
+
+     private function splitName($fullName) {
+
+          // Check if the full name is empty
+         if (empty($fullName)) {
+                return array(
+                    'first_name' => '',
+                    'last_name' => ''
+                );
+        }
+        // Explode the name by space
+        $nameParts = explode(' ', $fullName);
+
+        // Set first name and last name
+        $firstName = isset($nameParts[0]) ? $nameParts[0] : '';
+        $lastName = isset($nameParts[1]) ? $nameParts[1] : $nameParts[0]; // If space is not there, use the whole name as the last name
+
+        // Create an array with first name and last name
+        $nameArray = array(
+            'first_name' => $firstName,
+            'last_name' => $lastName
+        );
+
+        return $nameArray;
+    }
+
+
+
+    /**
+     * Check if a string starts with a specific substring.
+     *
+     * @param string $haystack The string to search in.
+     * @param string $needle   The substring to check for at the beginning.
+     *
+     * @return bool Whether the string starts with the specified substring.
+     */
+    public function startsWith($haystack, $needle)
+    {
+        // Check if the PHP version is 8 or above to use the built-in function
+        if (phpversion() >= 8) {
+            // Use str_starts_with function if PHP version is 8 or above
+            return str_starts_with($haystack, $needle);
+        }
+
+        // Use the traditional substr method for older PHP versions
+        return substr($haystack, 0, strlen($needle)) === $needle;
+    }
+
+    /**
+     * Get the formatted phone number with the correct extension based on the country code.
+     *
+     * @param string $code              The country code.
+     * @param array  $countryCodeArray  The array containing the country code information.
+     * @param string $phoneNumber       The original phone number.
+     *
+     * @return string The formatted phone number with the correct extension.
+     */
+    private function getPhoneExtension($code, $countryCodeArray, $phoneNumber)
```

Comment:
A lot of code has been written around working with phone numbers / switching between international + local.

It may be simpler to re-use the helpers already available e.g., `Utils::localPhoneToInternational()` or `Utils::internationalPhoneToEpp()` which under the hood use a port of google's libphonenumber

Reply:
$contactPhone = Utils::localPhoneToInternational($registrantDetails['phone'],$registrantDetails['country_code']); 
removed getPhoneExtension function from the code